projects/driver_controls/inc/ads1015_bitmasks.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+// ADS1015 I2C address definition (Datasheet Section 8.5.1.1)
+#define ADS1015_I2C_ADDRESS 0x48
+
+// ADS1015 I2C reset byte (Datasheet Section 8.5.1.2)
+#define ADS1015_RESET_BYTE 0x06
+
+// Register pointer definitions (Datasheet Table 4)
+#define ADS1015_CONVERSION_REGISTER 0x00
+#define ADS1015_CONFIG_REGISTER 0x01
+#define ADS1015_LO_THRESH_REGISTER 0x02
+#define ADS1015_HI_THRESH_REGISTER 0x03
+
+// ADS1015 Bitmasks (According to their positions in their data byte)
+#define ADS1015_CONFIG_OS(byte) (byte & 0x7F)
```

Comment:
Why not change these macros to take a pointer? When you invoke all of these with `byte = MACRO(byte);` it is confusing? `MACRO(&byte)` is a lot cleaner IMO?

Reply:
Please add brackets around macro arguments (ie. add brackets to wherever you use the argument in your macro)

```c
#define ADS1015_CONFIG_OS(byte) ((byte) & 0x7F)
```